app/components/form/fields/RadioField.tsx

```diff
@@ -168,6 +169,7 @@ export function RadioFieldDyn<
             aria-describedby={description ? `${id}-label-tip` : undefined}
             onChange={onChange}
             name={name}
+            disabled={disabled}
```

Comment:
You can probably get rid of these too. But not the one in `NumberField`, I guess.